### PR TITLE
[ci] fixes #73 - HW daemon pre-release

### LINKED ISSUE #73: Pre-release testing procedure for hw-daemon service
**Feature description**

Pre-release procedure

**Describe the solution you'd like**
Reproduce the pre-release steps used for hw-js library and adapt to using hw-go library commands.

**Additional context**
Needs the patch submitted in skycoin/skycoin#2396 for skycoin/skycoin#2395 .

**Possible implementation**

- [x] Copy pre-release steps from hw-js library, see skycoin/hardware-wallet-js#117 for skycoin/hardware-wallet-js#112
- [x] Replace references to hw-js functions with equivalent `skycoin-hw-cli` commands
- ~Copy and adapt hw-js pre-release CI scripts to execute same steps~

### PR DESCRIPTION
**Pull request by olemis**
Fixes #73

Changes:
- Pre-release testing procedure for hw-daemon . Adapted from hw-js ... with references to hw-js commands replaced with corresponding hw-daemon REST endpoints

Does this change need to mentioned in CHANGELOG.md?
no

Requires changes in protobuff specs?
no

Requires testing
no

### CODE REVIEW

**Comment by olemis** [comment]
@therealssj I can't tag you for review . Nonetheless , some changes might still be needed , and pre-release scripts added . Will add them shortly after .

File: README.md [hunk 1]
```diff
@@ -303,10 +303,223 @@ dep ensure github.com/foo/bar@tag
...
 #### Pre-release testing
 
-Performs these actions before releasing:
+Pre-release testing procedure requires [skycoin-cli](https://github.com/skycoin/skycoin/tree/develop/cmd/cli). Please [install it](https://github.com/skycoin/skycoin/blob/develop/cmd/cli/README.md#install) if not available in your system. Some operations in the process require [running a Skycoin node](https://github.com/skycoin/skycoin/tree/master/INTEGRATION.md#running-the-skycoin-node). Also clone [Skywallet firmware repository](https://github.com/skycoin/hardware-wallet/) in advance.
 
-* `make lint`
-* `make check` (make sure that all the integration tests are passing)
+The instructions that follow are meant to be followed for Skywallet devices flashed without memory protection. If your device memory is protected then some values might be different e.g. `firmware_features`.
+
+During the process beware of the fact that running an Skycoin node in the background can block the Skywallet from running.
```
Comment (therealssj) [inline README.md]: what do you mean it blocks skywallet ?

File: README.md [hunk 2]
```diff
@@ -303,10 +303,223 @@ dep ensure github.com/foo/bar@tag
...
+
+Some values need to be known during the process. They are represented by the following variables:
+
+- `WALLET1`, `WALLET2`, ... names of wallets created by `skycoin_cli`
```
Comment (therealssj) [inline README.md]: `skycoin-cli`

File: README.md (hunk 1 above)
Comment (olemis) [reply README.md]: It should be Skycoin desktop wallet ...

Comment (olemis) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (olemis) [reply README.md]: fixed

**Pull request merged**